build/params.go

```diff
@@ -60,7 +61,7 @@ const WRatioDen = 2
 // Proofs
 
 // Blocks
-const ProvingPeriodDuration = 160
+const ProvingPeriodDuration uint64 = 20
 
 // PoStChallangeTime sets the window in which post computation should happen
```

Comment:
I assume changes in this file are for testing.

Reply:
Will change back when deploying devnet (This will get merged into the next devnet branch)